src/App/Pages/Accounts/TwoFactorPageViewModel.cs

```diff
@@ -302,5 +311,17 @@ public async Task<bool> SendEmailAsync(bool showLoading, bool doToast)
                 return false;
             }
         }
+
+        public void SetEnableContinue(bool isTokenEmpty)
+        {
+            if (isTokenEmpty)
+            {
+                EnableContinue = false;
+            }
+            else if (!EnableContinue)
```

Comment:
Checking `!EnableContinue` before setting it to `true` is unnecessary. The compiler would probably optimize it out anyway. I think you could remove this method entirely and set `_vm.EnableContinue` directly from `Token_TextChanged`.

Reply:
Thanks @eliykat . I'll update the code over the weekend

Reply:
✅